progsrc/android-template/app/src/main/scala/com/audeering/opensmile/androidtemplate/plugins/Config.scala

```diff
@@ -26,10 +26,8 @@ object Config {
   val assets = List(
     "liveinput_android.conf",
     "BufferModeRb.conf.inc",
-    "BufferModeLive.conf.inc",
     "messages.conf.inc",
-    "music_features.conf.inc",
-    "FrameModeMoodWin.conf.inc"
+    "features.conf.inc"
```

Comment:
Android runtime error (over logcat): couldn't find the files `BufferModeLive.conf.inc`, `music_features.conf.inc`, `FrameModeMoodWin.conf.inc`. Also, the `features.conf.inc` file was not used anywhere. The app works as expected after the above modification